src/uwtools/resources/jsonschema/chgres-cube.jsonschema

```diff
@@ -197,6 +221,11 @@
                       "type": "string"
                     }
                   },
+                  "required": [
+                    "mosaic_file_target_grid",
+                    "varmap_file",
+                    "vcoord_file_target_grid"
```

Comment:
These were already looked up with `[]` in the driver code, so I've assumed that they are required and am enforcing that via this schema.

Reply:
According to the [UFS_UTILS docs](https://noaa-emcufs-utils.readthedocs.io/en/latest/ufs_utils.html#global-chgres-cube-namelist-options), `varmap_file` is only required for the grib2 input case. The other two in this required list are present in all three input format cases (grib2, nemsio, and netcdf). Perhaps we should remove that literal requirement here and in the code?

I did not follow up with a code deep-dive to confirm the docs are correct.

Reply:
I made `varmap_file` optional. Thanks for the fact check.